script.js

```diff
@@ -0,0 +1,40 @@
+var rooot = document.getElementById('root');
+var parentInp = document.createElement('input');
+var firstButton = document.createElement('button');
+var secondButton = document.createElement('button');
+var textMessage = document.createElement('p');
+var randomBgButton = document.createElement('button')
+var lastButton = document.createElement('button')
+parentInp.classList.add('texts');

```

Comment:
Adding a class will not be needed if you write HTML in a proper file with needed CSS selector.